### ISSUE TITLE
Run tests only in command line.

### ISSUE THREAD

**Issue by DamianRodziewicz** [mad-eye/meteor-mocha-web#124]
Hi guys,

We would like to use mocha to test our meteor app only using command line, when we directly run ```meteor --test```.

Is it possible to configure the package so that velocity does not start together with the app during development?

**Comment by Kestanous** [comment]
:+1:

**Comment by marekrogala** [comment]
:+1:

**Comment by marekrogala** [comment]
In other words, is it possible to have mocha integration with velocity using this package, but without forcing us to use html-reporter?

Do you think that splitting this package into mike:mocha-base and mike:mocha, with the latter simply depending on mocha-base and velocity:html-reporter would be a good solution?

**Comment by rissem** [comment]
That's been suggested before, and I think it's probably a good idea. I've been holding off, just because it means I have to release two packages instead of one. Would it be enough to be able to "turn off" the reporter, or are you concerned that the reporter is interfering with the front-end of your app?

**Comment by DamianRodziewicz** [comment]
Yes, it would be helpful to be able to turn the reporter off instead of changing css files. 

Splitting the package is a good idea. It would also be nice to have an ability to turn whole velocity off with environment variable. I tested meteor-jasmine and you can just write "VELOCITY=0 meteor" which switches velocity off for the run.

**Comment by Tolmark12** [comment]
:+1: for both the ability to turn off with an evironment var and a command line only version

**Comment by mvaivre** [comment]
yes please!

**Comment by brykov** [comment]
:+1: For passing environment variables to Velocity

**Comment by hellogerard** [comment]
Related to #101

**Comment by alpjor** [comment]
+1

**Comment by dmitrijs-balcers** [comment]
+1

**Comment by rista404** [comment]
:+1:

**Comment by came** [comment]
+1
